packages/cli/src/commands/login/future/index.ts

```diff
@@ -0,0 +1,201 @@
+import readline from 'node:readline';
+import chalk from 'chalk';
+import * as open from 'open';
+import type Client from '../../../util/client';
+import { getFlagsSpecification } from '../../../util/get-flags-specification';
+import { parseArguments } from '../../../util/get-args';
+import { printError } from '../../../util/error';
+import { help } from '../../help';
+import { loginCommand } from './command';
+import { updateCurrentTeamAfterLogin } from '../../../util/login/update-current-team-after-login';
+import {
+  writeToAuthConfigFile,
+  writeToConfigFile,
+} from '../../../util/config/files';
+import getGlobalPathConfig from '../../../util/config/global-path';
+import { getCommandName } from '../../../util/pkg-name';
+import { emoji, prependEmoji } from '../../../util/emoji';
+import hp from '../../../util/humanize-path';
+import {
+  deviceAuthorizationRequest,
+  processDeviceAuthorizationResponse,
+  deviceAccessTokenRequest,
+  processDeviceAccessTokenResponse,
+  isOAuthError,
+  verifyJWT,
+} from '../../../util/oauth';
+import o from '../../../output-manager';
+import { LoginTelemetryClient } from '../../../util/telemetry/commands/login';
+
+export async function login(client: Client): Promise<number> {
+  o.warn('This feature is under active development. Do not use!');
```

Comment:
Do we need this? Can we not allow for users to begin using it right away?

Reply:
Will drop. :+1: